plugins/inputs/chrony/chrony.go

```diff
@@ -20,22 +20,30 @@ var (
 )
 
 type Chrony struct {
-	path string
+	DNSLookup bool `toml:"dns_lookup"`
+	path      string
 }
 
 func (*Chrony) Description() string {
 	return "Get standard chrony metrics, requires chronyc executable."
 }
 
 func (*Chrony) SampleConfig() string {
-	return ""
+	return `
+  ## If true, chronyc tries to perform a DNS lookup for the time server.
+  # dns_lookup = false
+  `
 }
 
 func (c *Chrony) Gather(acc telegraf.Accumulator) error {
 	if len(c.path) == 0 {
 		return errors.New("chronyc not found: verify that chrony is installed and that chronyc is in your PATH")
 	}
-	cmd := execCommand(c.path, "tracking")
+	flags := []string{"-n", "tracking"}
```

Comment:
seems more explicit to _add_ flags to the array if you want them, rather than remove when you don't.

I would change to:

```
flags := []string{}
if !c.DNSLookup {
  flags = append(flags, "-n")
}
flags = append(flags, "tracking")
```

```